tests/appsec/iast/test_security_controls.py

```diff
@@ -139,6 +140,7 @@ def setup_no_vulnerability_suppression_with_a_sanitizer_configured_for_an_overlo
         self.setup_iast_is_enabled()
         self.r = weblog.post("iast/sc/s/overloaded/insecure", data={"param": "param"})
 
+    @irrelevant(context.library == "nodejs")
```

Comment:
Non blocking comment : it can be useful for later readers to add a `reason` arg explaining why it's not relevant.